packages/odata-common/src/edm-types.ts

```diff
@@ -23,18 +24,17 @@ export type EdmTypeSameConverters =
   | 'Edm.Any'; // Represents currently unsupported EDM types like Edm.Geography.
 
 /**
- * @internal
+ * PLEASE REVIEW.
  */
 export type EdmTypeDifferentConverters = 'Edm.DateTimeOffset';
 
-// Exclusive types for ODataVersions
 /**
- * @internal
+ * Exclusive types for OData v2.
```

Comment:
[q] Why does this have to be exported?

```suggestion
 * Exclusive EDM types for OData v2.
```

Reply:
Warning: ExclusiveEdmTypeV2, defined at packages/odata-common/dist/edm-types.d.ts:17, is referenced by @sap-cloud-sdk/odata-common.EdmTypeShared but not included in the documentation.

And the shared type is used in all packages.